pkg/operator/operator.go

```diff
@@ -219,9 +219,6 @@ func NewOperator() (context.Context, *Operator) {
 		return []string{o.(*v1.NodePool).Spec.Template.Spec.NodeClassRef.Name}
 	}), "failed to setup nodepool nodeclassref name indexer")
 
-	lo.Must0(mgr.AddReadyzCheck("manager", func(req *http.Request) error {
```

Comment:
It's not clear to me what the tangible impact of this change is -- this was obviously added for a reason when it was first introduced and I'm cautious of removing it when we know we are just going to add it back in as soon as we drop the webhooks.

Reply:
Synced offline